Userland/Applications/Browser/DownloadWidget.cpp

```diff
@@ -39,7 +40,8 @@ DownloadWidget::DownloadWidget(const URL& url)
     auto close_on_finish = Config::read_bool("Browser", "Preferences", "CloseDownloadWidgetOnFinish", false);
 
     m_elapsed_timer.start();
-    m_download = Web::ResourceLoader::the().protocol_client().start_request("GET", url);
+    auto protocol_client = Protocol::RequestClient::try_create().release_value_but_fixme_should_propagate_errors();
```

Comment:
This causes every download to create a new IPC client, doesn't it? that seems less than ideal

Reply:
Switched it to use the new `Web::ResourceLoader::the()` singleton (and giving said singleton more capabilities as well)